tsc/OpenTimelineIO_TSC_Charter.md

```diff
@@ -0,0 +1,67 @@
+# Technical Charter (the "Charter") for OpenTimelineIO a Series of LF Projects, LLC
+
+This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenTimelineIO project, which has been established as OpenTimelineIO a Series of LF Projects, LLC (the "Project"). LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
+
+## 1. Mission and Scope of the Project
+  * **a.** The mission of the Project is to develop an open source project with the goals indicated in the "README" file within the Project's repositories.
+  * **b.** The scope of the Project includes software development under an OSI-approved open source license supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source software project.
+
+## 2. Technical Steering Committee
+  * **a.** The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
+  * **b.** The TSC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
+  * **c.** TSC projects generally will involve Contributors and Committers. The TSC may adopt or modify roles so long as the roles are documented in the CONTRIBUTING file. Unless otherwise documented:
+    * **i.** Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project;
```

Comment:
```suggestion
    * **i.** Contributors include anyone in the technical community that contributes code, documentation, or other artifacts to the Project;
```
A piece of paper with a logo drawing on it is a non-technical artifact that could be contributed to the project. It may be scanned, producing a technical artifact but the piece of paper itself would be the originating contribution.

Reply:
I don't think the word "technical" is trying to distinguish physical vs digital. Though, the project can only accept digital contributions in practice. I think the word "technical" is meant to match the meaning of "Technical Steering Committee" which is that the TSC makes technical decisions for a technology project.

Reply:
Yes, that makes sense to me.